webpack.config.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
+const StyleLintWebpackPlugin = require('stylelint-webpack-plugin');
```

Comment:
Shouldn't it be used somewhere in the config?

Reply:
should be removed